line/parseData.js

```diff
@@ -22,26 +22,27 @@ export function load(url, options) { // eslint-disable-line
 
         // Automatically calculate the seriesnames excluding the "marker" and "annotate column"
         const seriesNames = getSeriesNames(data.columns);
-
         // Use the seriesNames array to calculate the minimum and max values in the dataset
         const valueExtent = extentMulti(data, seriesNames, yMin);
 
         // Format the dataset that is used to draw the lines
-        const plotData = seriesNames.map(d => ({
+        let highlightLines = {};
+        let plotData = seriesNames.map(d => ({
             name: d,
             lineData: getlines(data, d),
+            highlightLine: getHighlightNames(d),
         }));
 
-        // Sort the data so that the labeled items are drawn on top
-        const dataSorter = function dataSorter(a, b) {
-            if (highlightNames.indexOf(a.name) > highlightNames.indexOf(b.name)) {
-                return 1;
-            } else if (highlightNames.indexOf(a.name) === highlightNames.indexOf(b.name)) {
-                return 0;
-            }
-            return -1;
-        };
-        if (highlightNames.length > 0) { plotData.sort(dataSorter); }
+        function getHighlightNames(el) {
+            let status = false;
+            highlightNames.map((d) => {
```

Comment:
`Array.prototype.forEach` probably makes more sense here.

Reply:
Confused as this whole function should no longer be there

Reply:
![title-not-yet-added](https://user-images.githubusercontent.com/4738610/33180756-afa87ffe-d065-11e7-8093-cf27336005bb.png)
This feature only kicks in whem you add values to the highlightNames array on the index.js. It makes all the other lines grey and only colours these. For when you have lots of lines but only a couple a relevant

Reply:
Actually, `getHighlightNames` is a bit confusing because it ultimately returns a boolean but the function implies it returns some other primitive (i.e., a string). Maybe it can be rewritten to something like...:
```js
        const isLineHighlighted = (el) => highlightNames.some(d => d === el);
```

[Array.prototype.some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)